swagger.yaml

```diff
@@ -5012,6 +5012,8 @@ definitions:
         description: Additional information about disks, formatted as JSON.
       role:
         $ref: '#/definitions/host-role'
+      suggested_role:
+        $ref: '#/definitions/host-role'
```

Comment:
This will only be master or worker, right?  Not auto-assign or bootstrap.

Reply:
I wish to save the endless conversions between role and suggested-role and wrote as much in the design.Apart from bootstrap suggested-role has the same values as role